dev/NavigationView/NavigationView.idl

```diff
@@ -108,6 +108,20 @@ runtimeclass NavigationViewBackRequestedEventArgs
 {
 }
 
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+runtimeclass NavigationViewExpandingEventArgs
+{
+	NavigationViewItemBase ExpandingItemContainer { get; };
```

Comment:
>NavigationViewItemBase [](start = 1, length = 22)

Should it be NavigationViewItem? I think Header and seperator should not be the container

Reply:
I believe @YuliKl wanted to keep this as NavigationViewItemBase for future-proofing

Reply:
Agreed that header and separator don't apply here, so NVI is the only relevant type that exists today. But I can envision us needing to add, maybe, a toggle item, or some other specialized item type that the args should continue to support in the future. Using NavigationViewItemBase feels like an easy way to not block functionality we haven't thought of yet :)